python/src/phylanx.cpp

```diff
@@ -497,36 +535,390 @@ PYBIND11_MODULE(_phylanx, m)
         "generate expression tree from given expression");
 
     execution_tree.def("var",
-        [](double d)
-        {
-            return hpx::threads::run_as_hpx_thread([&]()
-                {
-                    using namespace phylanx::execution_tree;
-                    return primitive{
-                        hpx::local_new<primitives::variable>(
-                            phylanx::ir::node_data<double>{d})};
-                });
+        [](double d) {
+            return hpx::threads::run_as_hpx_thread([&]() {
+                using namespace phylanx::execution_tree;
+                return primitive{hpx::local_new<primitives::variable>(
+                    phylanx::ir::node_data<double>{d})};
+            });
         },
         "create a new variable from a floating point value");
 
+    execution_tree.def("linspace",
+        [](double xmin, double xmax, std::size_t nx) {
```

Comment:
What does 'linspace' stand for?

Reply:
linspace() is a replication of a function in numpy that creates an array with nx values ranging from xmin to xmax.